core/controllers/admin_test.py

```diff
@@ -609,6 +643,7 @@ def test_stop_computation_with_running_jobs(self):
         self.logout()
 
     def test_stop_computation_with_finished_jobs(self):
+        """ Test that Stop Computation with Finished Jobs. """
```

Comment:
We don't need a docstring for the setUp and test function, can you please remove those docstrings?

(I've asked contributors to update the issue description in #11776, sorry for the trouble)

test function = function name starting with `test_`

Reply:
> We don't need a docstring for the setUp and test function, can you please remove those docstrings?
> 
> (I've asked contributors to update the issue description in #11776, sorry for the trouble)
> 
> test function = function name starting with `test_`

almost all have `test_` then what have to do in this file?